### PR TITLE
readme

### PR DESCRIPTION
**Pull request by ncarignan**

### CODE REVIEW

File: class-01/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Class 01: SMACSS and Responsive Web Design
+
```
Comment (crazyWilliams) [inline class-01/README.md]: ITS JUH BOIIIIIII

Comment (jenwill) [inline class-01/README.md]: Jennifer was here!

File: class-01/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Overview
+
+Today we will be covering some new topics and reviewing concepts you should already be familiar with. For students who completed Code 201, today should be a review of CSS floats.
```
Comment (MSpake) [inline class-01/README.md]: Yay!

**Comment by ncarignan** [comment]
we need a nap (denevan)

File: class-01/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+We will spend some time reviewing the concepts from the lab 00 assignments. We will also discuss several new and very important topics, including state, scope, and the "new" keyword.
+
+## Daily Plan
+
```
Comment (ncarignan) [inline class-01/README.md]: `taking a nap`

**Pull request merged**